### ISSUE TITLE
BPMN/DMN palette is not focusable

### ISSUE THREAD

**Issue by lmbateman** [camunda/camunda-modeler#5687]
### Describe the bug

When tabbing through the BPMN or DMN editor, the palette never receives focus.

### Steps to reproduce

1. Open a BPMN or DMN diagram.
2. Tab through the page until the canvas is focused.
3. Press Tab again. Based on the visual layout of the page, the logical next place for focus should be the palette. Instead, it's the first item in the context pad (WM BPMN) or the icon on the decision (WM DMN) or the first focusable item in the properties panel (DM)

### Expected behavior

After the canvas, focus should land on the palette, then the context pad (if open).

## Camunda Modeler system information
 * Version: 5.44.0
 * Operating System: Mac OS 10.15.7 
 * Plugins: bpmn-js Token Simulation
 * Execution Platform: BPMN - Camunda 8

### Additional context

Web Modeler team [requested support from the Core Modeling team](https://github.com/camunda/web-modeler/issues/21204#issuecomment-3943531132):

> We might need support from the bpmn-io team here; it's the second ticket that I encountered about focus interaction between WM and the canvas, and I fear we might need an overarching strategy to tackle these cases.
> 
> E.g. hand it over to the bpm-io team? Come up with a focus manager on the WM who would take care of these cases? Sounds like we need a technical assessment before deciding what to do.

**Comment by nikku** [comment]
Why would you want to access the tabbing via keyboard, if you can press `c` or `a` to create/append any element?

![Image](https://github.com/user-attachments/assets/b8628bc3-17de-473a-9947-54c3eac18fc4)

What is our strategy?

**Comment by jarekdanielak** [comment]
@lmbateman would you like to follow up on this?

**Comment by lmbateman** [comment]
Good question. I'm not sure that making the palette reachable by keyboard is sufficient; once they land there, users will want to be able to DO something. So we probably need to figure out how to make the palette fully interactive by keyboard, which is a larger project (and means the title of this issue is misleading). We've had at least one customer request to do just that, but the design team is about to do a deep dive into what our customers really need, accessibility-wise, and it's possible full palette accessibility won't make the prioritization cut, at least for 8.10. I see a couple of options:
1. Close this issue for now. I can always open another one later when it's more pressing.
2. Keep the issue open, with a new name and description to reflect the actual scope, and I can add some ideas for how to approach it from a UX perspective.

Either way, it might make sense to have a separate issue to review and update the keyboard shortcut documentation and dialogs, especially in Web Modeler. For example, I see `c` in the relevant application menu in Desktop Modeler, but not in the dialog in Web Modeler, plus I can't figure out how it's supposed to work, and kapa.ai doesn't know about it at all.

**Comment by barmac** [comment]
Thanks for explanation. So this issue boils down to keyboard-only modeling which should be covered via https://github.com/camunda/camunda-modeler/issues/5688 if we decide to implement that. Focusable palette without the next step does not enable the user to do anything, so we won't implement that on its own. Because of that, I will close this issue.
